component-form/component-form-core/src/main/java/org/talend/sdk/component/form/internal/converter/impl/widget/AbstractWidgetConverter.java

```diff
@@ -172,6 +172,9 @@ protected UiSchema newOrphanSchema(final PropertyContext<?> ctx) {
         if (Boolean.parseBoolean(ctx.getProperty().getMetadata().getOrDefault("documentation::tooltip", "false"))) {
             schema.setTooltip(ctx.getProperty().getMetadata().get("documentation::value"));
         }
+        if (Boolean.parseBoolean(ctx.getProperty().getMetadata().getOrDefault("ui::readonly", "false"))) {
+            schema.setReadOnly(true);
```

Comment:
wow, we had this property???

Reply:
It's in the UiSchema specification so already existed.